conf/upf.json

```
@@ -61,12 +63,14 @@
 
     "": "Gateway interfaces",
     "access": {
-        "ifname": "ens803f2"
+        "ifname": "ens803f2",
+	"cndp_jsonc_file": "/opt/bess/bessctl/conf/cndp_upf_1worker.jsonc"
     },
 
     "": "UE IP Natting. Update the line below to `\"ip_masquerade\": \"<ip> [or <ip>]\"` to enable",
     "core": {
         "ifname": "ens803f3",
+	"cndp_jsonc_file": "/opt/bess/bessctl/conf/cndp_upf_1worker.jsonc",
```

Please use spaces instead of `tab`
```suggestion
        "cndp_jsonc_file": "/opt/bess/bessctl/conf/cndp_upf_1worker.jsonc",
```